src/commands/ticker/index/processor.ts

```diff
@@ -470,7 +472,8 @@ export function parseQuery(query: string) {
 
 export async function ticker(
   msgOrInteraction: Message | CommandInteraction,
-  base: string
+  base: string,
+  chain: string
```

Comment:
fallback ở đây vì cả 2 text/slash đều gọi function này

```suggestion
  chain = ""
```